### DISCUSSION TITLE
Can add these solvers?

### DISCUSSION THREAD

**Discussion by rastarr** [FlareSolverr/FlareSolverr#677] [Q&A]
Hi,
Just new here and great to see this inside a docker image
From what I gather, there are no working (and free) captcha solvers.

Is it possible to implement the following solvers?

https://github.com/QIN2DIM/hcaptcha-challenger
https://github.com/QIN2DIM/recaptcha-challenger

**Comment by ngosang** [comment]
We already have some issues for solvers. PRs are welcome.
https://github.com/FlareSolverr/FlareSolverr/issues?q=is%3Aissue+is%3Aopen+label%3A%22captcha+solver+service%22
